release/1.1.1

mcgalcode opened the pull request:
Bugfix to resolve paths getting mangled during recursive ingest.

maxhutch committed:
Merge pull request #33 from CitrineInformatics/release/1.1.0

Resync develop

mcgalcode committed:
Fix Typo In setup.py

mcgalcode committed:
Bump PyCC Dependency For Functional Upload

  Bumps the version of PyCC to take advantage of functional
  upload tooling in the newest version. Specifically, this
  fixes the bug where nested directories were appearing in
  Citrination with mangled paths.

mcgalcode committed:
Update requirements.txt to reflect new pycc dep

mcgalcode committed:
Bump pypif sdk version

maxhutch committed:
Merge pull request #34 from CitrineInformatics/bugfix/nested-dirs

bugfix - nested directory upload

mcgalcode committed:
Bump PyCC Version To Fix Nested Directories Bug

maxhutch committed:
Merge pull request #35 from CitrineInformatics/bugfix/update-pycc

Bump PyCC Version To Fix Nested Directories Bug

mcgalcode committed:
Version Bump

maxhutch commented on requirements.txt:
```diff
@@ -1,5 +1,5 @@
 pypif==2.0.1
-pypif_sdk==2.0.0
-citrination_client==2.1.0
+pypif_sdk>=2.2.1,<3
```
These should be pinned in `requirements.txt` and open (like this) in setup.py (as they are).

mcgalcode committed:
Pin Versions In Requirements.txt

  per recommendation here:

  https://packaging.python.org/discussions/install-requires-vs-requirements/#requirements-files
